quarkus/runtime/src/main/resources/META-INF/keycloak.conf

```diff
@@ -1,5 +1,5 @@
 # Default and non-production grade database vendor
-db=dev-file
+%dev.db=dev-file
```

Comment:
We do need to make this change now, do we? We're not changing the behaviour yet.

Reply:
We don't have to make this change, but it's effectively redundant to have db=dev-file here and as the default on the option. Since we're making other changes in preparation for the breaking change, I would make this one as well - that will leave just updating the option later.

Reply:
Ok, let's keep it then.